examples/todoapp/README.md

```diff
@@ -1,6 +1,6 @@
-An example of Kotlin Multiplatform todo app with shared Jetpack Compose UI.
+An example of Kotlin Multiplatform todo app with shared Android/Desktop Compose UI and SwiftUI iOS.
```

Comment:
An example of Kotlin Multiplatform todo app with shared Android/Desktop Compose UI and SwiftUI (not Compose) iOS.